components/admin_console/team_channel_settings/team/details/team_details.jsx

```diff
@@ -128,7 +128,7 @@ export default class TeamDetails extends React.Component {
                 filter((g) => {
                     return !origGroups.some((group) => group.id === g.id);
                 }).
-                map((g) => actions.linkGroupSyncable(g.id, teamID, Groups.SYNCABLE_TYPE_TEAM, {auto_add: true}));
+                map((g) => actions.linkGroupSyncable(g.id, teamID, Groups.SYNCABLE_TYPE_TEAM, {auto_add: true, scheme_admin: g.scheme_admin}));
```

Comment:
We should add some tests for the link/unlink/patch logic

Reply:
Done! Thanks